README.md

```diff
@@ -16,14 +16,18 @@ npm install
 After completing above installation instructions, to run locally without Basic Authentication:
 
 ``` 
-DEBUG=rubbernecker:* PIVOTAL_API_KEY=your-api-key PIVOTAL_PROJECT_ID=your-project-id npm start
+DEBUG=rubbernecker:* PIVOTAL_API_KEY=your-api-key PIVOTAL_PROJECT_ID=your-project-id PAGERDUTY_API_TOKEN=your-pagerduty-token npm start
 ```
 
 If you want to try out the stunning Basic Auth (useful for when you don't want everyone getting straight to your story overviews)
 ``` 
-DEBUG=rubbernecker:* PIVOTAL_API_KEY=your-api-key PIVOTAL_PROJECT_ID=your-project-id USE_AUTH=true USERNAME=username PASSWORD=password npm start
+DEBUG=rubbernecker:* PIVOTAL_API_KEY=your-api-key PIVOTAL_PROJECT_ID=your-project-id PAGERDUTY_API_TOKEN=your-pagerduty-token USE_AUTH=true USERNAME=username PASSWORD=password npm start
 ```
 
+#### PagerDury Support
+
+Rubbernecker supports PagerDuty, for pulling out the current rota information. This requires a `PAGERDUTY_API_TOKEN` environment variable, to be passed before the app is run.
```

Comment:
It does. It's wasn't very clear to me as why... But essentially, if !`(!error && response.statusCode == 200)` it will fall through to returning a null instead of an error - being graceful.

https://github.com/alphagov/paas-rubbernecker/pull/23/files#diff-add52c09e06294794ef504bd4124bcbfR182